docs/DeveloperGuide.md

```diff
@@ -523,6 +523,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ****
 
+**Use case 9: Find food stalls**
+
+**MSS**
+
+1. User starts FoodWhere.
+2. User enters the command to find food stalls.
+3. FoodWhere displays the food stalls that matches the search keywords.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. User uses at least one of the prefixes as a search term:
+    * `n/` for Name search.
```

Comment:
I think exact prefixes are too specific for use case.

Reply:
Fixed.